modules/httpcache/httpcache_test.go

```diff
@@ -24,6 +25,17 @@ func (m mockFileInfo) ModTime() time.Time { return time.Time{} }
 func (m mockFileInfo) IsDir() bool        { return false }
 func (m mockFileInfo) Sys() interface{}   { return nil }
 
+func countFormalHeaders(h http.Header) (c int) {
+	for k := range h {
+		// ignore our headers for internal usage
+		if strings.HasPrefix(k, "X-Gitea-") {
```

Comment:
I wouldn't even check for `X-Gitea`,
I would instead check only for `X-` as according to my knowledge any header starting with that is not standardized.
You do not know if there will ever be a custom header that is not `X-Gitea`.

Oh wait, I just noticed that this is a testing method where you can indeed influence what headers are being sent.
I guess you can then ignore this comment.

Reply:
Yep, explained above.